tests/test_habitat_quality.py

```diff
@@ -1316,12 +1315,12 @@ def test_habtitat_quality_validation_missing_sens_header(self):
                 '0.07,1.0,threat_2,exponential,,threat_2_c.tif,'
                 'threat_2_f.tif\n')
 
+        # At least one threat header is expected, so there should be a message
         validate_result = habitat_quality.validate(args, limit_to=None)
-        self.assertTrue(
-            validate_result,
-            "expected failed validations instead didn't get any.")
-        self.assertTrue(
-            'any column in the sensitivity table' in validate_result[0][1])
+        expected = [(
+            ['sensitivity_table_path'],
+            '(?!lulc|habitat|name)(^.+$) matched 0 headers, expected at least one')]
```

Comment:
I don't think user-facing validation messages should include regular expressions. Can we think of a way to return a more readable message?

Reply:
see comment https://github.com/natcap/invest/pull/526#discussion_r652766326